models/api_key.go

```diff
@@ -0,0 +1,14 @@
+package models
+
+type ApiKey struct {
```

Comment:
We now have API keys stored in two different places. There is the "main" API key as part of the user model and additional API keys in form of this many-to-many relation. I think that's somewhat confusing both for a user and for us as developers. What do you think?